plugins/CuraEngineBackend/CuraEngineBackend.py

```diff
@@ -30,11 +31,12 @@
 from UM.i18n import i18nCatalog
 catalog = i18nCatalog("cura")
 
-class CuraEngineBackend(Backend):
+class CuraEngineBackend(QObject, Backend):
     ##  Starts the back-end plug-in.
     #
     #   This registers all the signal listeners and prepares for communication
     #   with the back-end in general.
+    #   CuraEngineBackend is exposed to qml as well.
     def __init__(self):
```

Comment:
I was under the impression that for QObjects you always need to set the parent. This is what we do for every other case where we use QObject. @awhiemstra Could you chime in on this?